usd/include/sdf/usd/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+ign_install_all_headers(COMPONENT usd)
```

Comment:
it looks like this is repeated in the `usd/include/sdf` and `usd/include` directories, but I don't think we need to have this file repeated in those locations - looking at how components are structured in `ign-common`, I believe things should work fine with this file only (i.e., we can keep the `CmakeLists.txt` file in `usd/include/sdf/usd` and delete the `CMakeLists.txt` files in the other include dirs).

Reply:
Removed the extra unneeded CMakeLists.txt file in https://github.com/ignitionrobotics/sdformat/pull/817/commits/45457236160a17e7f632d337ce0e9de4f91e2147